client/src/components/ProfileSettings/Options/Options.tsx

```diff
@@ -19,22 +30,66 @@ const Options = (): JSX.Element => {
       anchor="left"
     >
       <List>
-        <ListItem button disableGutters component={Link} to="/profile/options/edit-profile">
+        <ListItem
+          button
+          disableGutters
+          selected={selectedIndex === 0}
+          onClick={(event: React.MouseEvent<HTMLElement>) => handleListItemClick(event, 0)}
+          component={Link}
+          to="/profile/options/edit-profile"
+        >
           <ListItemText primary="Edit Profile" className={classes.listItem} />
         </ListItem>
-        <ListItem button disableGutters component={Link} to="/profile/options/profile-photo">
+        <ListItem
+          button
+          disableGutters
+          selected={selectedIndex === 1}
+          onClick={(event: React.MouseEvent<HTMLElement>) => handleListItemClick(event, 1)}
+          component={Link}
+          to="/profile/options/profile-photo"
+        >
           <ListItemText primary="Profile Photo" className={classes.listItem} />
         </ListItem>
-        <ListItem button disableGutters component={Link} to="/profile/options/availability">
-          <ListItemText primary="Availability" className={classes.listItem} />
-        </ListItem>
-        <ListItem button disableGutters component={Link} to="/profile/options/payment">
+        {testType === ProfileType.Sitter && ( // TODO: This condition will be modified accordingly once integration is applied
```

Comment:
I really like these comments - as the project grows and the structure gets more and more complicated, these will come in clutch!